contracts/StablecoinConverter.sol

```diff
@@ -156,7 +156,7 @@ contract StablecoinConverter is EpochTokenLocker {
         uint16[] tokenIdsForPrice;
         address solutionSubmitter;
         uint256 feeReward;
-        uint256 objectiveValue;
+        uint objectiveValue;
```

Comment:
I liked uint256 better :)

Reply:
Shouldn't this be an int? Are we sure disregarded utility can never be larger than utility?

Reply:
I started with that, but the argument (which must have been offline) went as follows, the unsafty introduced with int is not worth the nuisance and since the trivial solution evaluates to zero, there is no point in having anything which evaluates below. 

Also notice the require statement below which enforces `utility > disregardedUtility`.

Until now, I have only managed to construct truly crappy, trivial,  examples whose objective is negative. For example, using the basicTrade consisting of two orders from the unit tests. One must fulfill 70% of each order\s requested buy amount in to meet the positivity threshold from our conditions. This suggests that fulfilment of these orders below 70% is considered a "worse solution" by our metric. 

Of course this is not exactly the ideal scenario for our objective function, but it does appear to capture quite a bit.

Reply:
I'm convinced it's ok to be a unsigned integer, but I also agree that it should make the width explicit and thus stay an uint**256**